torch/csrc/jit/script/init.cpp

```diff
@@ -246,14 +246,15 @@ struct ModuleValue : public SugaredValue {
     }
     throw ErrorReport(loc) << "module has no attribute '" << field << "'";
   }
+
   // call module.forward
   virtual std::shared_ptr<SugaredValue> call(SourceRange loc, Method & caller, at::ArrayRef<NamedValue> inputs, at::ArrayRef<NamedValue> attributes, size_t n_binders) override {
     return attr(loc, caller, "forward")->call(loc, caller, inputs, attributes, n_binders);
   }
 
   virtual std::vector<std::shared_ptr<SugaredValue>> asTuple(SourceRange loc, Method& m) override {
     py::object py_module = py::cast(module);
-    if(!py::isinstance(py_module, py::module::import("torch.jit").attr("_ConstModuleList")))
+    if(!py::isinstance(py_module, py::module::import("torch.jit").attr("_ConstModuleList")) && !py::isinstance(py_module, py::module::import("torch.jit").attr("_ConstSequential")))
```

Comment:
It would be nice to refactor this to a separate function, that will also import `torch.jit` once for both checks.